gio/src/subclass/application.rs

```diff
@@ -110,6 +110,10 @@ pub trait ApplicationImpl:
     fn handle_local_options(&self, options: &VariantDict) -> ExitCode {
         self.parent_handle_local_options(options)
     }
+
+    fn dbus_register(&self, connection: &DBusConnection, object_path: &str) -> Result<bool, Error> {
```

Comment:
This should return `Result<(), Error>` AFAICS. The C function is supposed to either return `FALSE` and set the error, or return `TRUE` and not set the error.

Reply:
Thanks, I'll change this. I must admit that I didn't entirely understand the semantics here 😇

Reply:
Me neither, the documentation was confusing and I had to look at the implementation :)

Reply:
😅